bfv/keygen.go

```diff
@@ -1,254 +1,36 @@
 package bfv
 
 import (
-	"math/big"
-
-	"github.com/ldsec/lattigo/v2/ring"
 	"github.com/ldsec/lattigo/v2/rlwe"
-	"github.com/ldsec/lattigo/v2/utils"
 )
 
 // KeyGenerator is an interface implementing the methods of the keyGenerator.
 type KeyGenerator interface {
 	GenSecretKey() (sk *rlwe.SecretKey)
```

Comment:
Can probably be rewritten as something like:

```
type KeyGenerator interface {
	rlwe.KeyGenerator
        <specific methods>
}
```

Reply:
fixed